src/mixxxapplication.cpp

```diff
@@ -41,13 +41,23 @@ Q_IMPORT_PLUGIN(qtaccessiblewidgets)
 #endif
 #endif
 
+namespace {
+
+class QMouseEventEditable : public QMouseEvent {
```

Comment:
Such a *transmutation* with the associated static_casts is a very dirty trick. Doing this without any explanation and an explicit warning is negligent.